### ISSUE TITLE
AGE calculation

### ISSUE THREAD

**Issue by kylebmetrum** [metrumresearchgroup/OpenBoneMin#5]
@riggsmm 

https://github.com/metrumresearchgroup/OpenBoneMin/blob/master/inst/OpenBoneMin.cpp#L634

```c
double AGE = ageENTER + T_0;
```

should be ??

```c
double AGE = ageENTER + SOLVERTIME;
```

**Comment by riggsmm** [comment]
Good catch -- mis-translated from the Berkeley-Madonna original code ("Age = startAge + TIME/(24*365.25)") as posted in .mmd file in online Supporting Information (psp4201210-sup-0004.zipZip archive, 17.3 KB	Supplementary Data 2) available at: https://ascpt.onlinelibrary.wiley.com/doi/abs/10.1038/psp.2012.10
